src/forecast.jl

```diff
@@ -0,0 +1,31 @@
+export Forecast, 
+    Scenarios
+
+struct Forecast
+
+end
+
+struct Scenarios{T <: Real}
+    name::String
+    timestamps::Vector{DateTime}
+    vals::Matrix{T}
+
```

Comment:
Pensei em de repente colocar o "name" dentro da struct de Forecast e retirar da de cenários.
Olhando o formato de entrada/saída, a struct de Forecast irá sempre ter a struct de cenários na entrada, e terá na de saída apenas se for uma previsão probabilística. Acho que talvez fique mais claro colocar então "name" na struct mais externa (forecast no caso). O que acha?

Reply:
Acho que poderiamos fazer duas structs, uma de Forecast e outra de cenários, a de Forecast pode ser mais completa

Reply:
Mas o nome ficaria repetido duas vezes mesmo então?

Reply:
eu pensei na struct de Forecast assim
```julia
struct Forecast{T <: Real}
    name::String
    timestamps::Vector{DaateTime}
    expected_vals::Vector{T}
    quantiles::Vector{T}
    confidence_intervals::Matrix{T}
```

Reply:
Entendi. Melhor assim mesmo do que em relação ao que eu tava pensando.. Acho que não conseguiria ser tão genérica quanto eu tava pensando.